src/js/views/settings/BlockedSettings.js

```diff
@@ -0,0 +1,53 @@
+import { html } from 'htm/preact';
+import _ from 'lodash';
+import State from '../../State';
+import Component from '../../BaseComponent';
+import {translate as t} from '../../Translation';
+import Session from '../../Session';
+
+export default class BlockedSettings extends Component {
+  constructor() {
+    super();
+    this.state = Session.DEFAULT_SETTINGS;
+    this.state.webPushSubscriptions = {};
+    this.state.blockedUsers = {};
+    this.id = "settings";
+  }
+  render() {
+    const blockedUsers = _.filter(Object.keys(this.state.blockedUsers), user => this.state.blockedUsers[user]);
+    
+    return (
+        <>
+        <div class="centered-container">
+        <h3>{t('blocked_users')}</h3>
+        {blockedUsers.map(user => {
+          if (this.state.blockedUsers[user]) {
+            return html`<p><a href="/profile/${encodeURIComponent(user)}"><${Text} user=${user} path="profile/name" placeholder="User"/></a></p>`;
```

Comment:
I previously used htm/preact templates: ``html`<${Text} />` ``. However, this is .jsx code and `<Text />` should be used instead. Same goes for variables like `user` here.